undebt/pattern/interface.py

```diff
@@ -78,10 +84,23 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        path = module_name_to_path(path)
```

Comment:
I think this should be `imp.find_module`? Otherwise you'll only be able to "find" modules in the current directory.